redisraft.c

```diff
@@ -651,7 +651,7 @@ static int cmdRaftShardGroup(RedisModuleCtx *ctx, RedisModuleString **argv, int
         }
 
         req = RaftReqInit(ctx, RR_SHARDGROUP_ADD);
-        if (ShardGroupParse(ctx, &argv[2], argc - 2, &req->r.shardgroup_add) != RR_OK) {
+        if (ShardGroupParse(ctx, &argv[2], argc - 2, &req->r.shardgroup_add, 2) < 0) {
```

Comment:
no longer an RR_STATUS, now it return -1 on error, or >= 0 (with number of args it consumed as the result) on success.  In practice, it might even >= 3 alwaysy